src/components/App/index.js

```diff
@@ -4,41 +4,28 @@ import './index.css';
 import Container from 'react-bootstrap/Container'
 import Form from '../Form';
 import ListWrapper from '../ListWrapper';
-import config from '../../config.json';
 
 export default function App () {
-  const {
-    categories,
-    categoryDisplayTitles,
-    filters,
-    filterDisplayTitles,
-    offerString,
-    requestString,
-    responseEmail
-  } = config;
-  const path = window.location.pathname;
+  let path = window.location.pathname;
+  // ensure any other paths just return home
+  if (path !== '/submit' && path !== '/') {
+    window.history.replaceState(null, '', '/');
+    path = '/';
+  }
 
   return (
     <Container className="App">
       <header className="border-bottom mb-3 mt-3">
         <h1>LMCC Needs/Provides Forum</h1>
-        <p className="lead">A place for the <a className="info" href="https://www.lowermanhattanchurch.com/">Lower Manhattan Community Church</a> community to share and meet our needs</p>
+        <p className="lead">A place for the <a className="info" href="https://www.lowermanhattanchurch.com/" target="_blank" rel="noopener noreferrer">Lower Manhattan Community Church</a> community to share and meet our needs</p>
```

Comment:
I think we can leave off `noreferrer` in this case. It is a church website to another church website. I don't _think_ there nearly as many security concerns in this case.